src/main/java/cz/fi/muni/pa165/esports/dao/MatchRecordRecordDaoImpl.java

```diff
@@ -10,9 +10,7 @@
 import javax.persistence.PersistenceContext;
 import java.util.List;
 
-/*
-    @Author Elena Álvarez
-*/
+
```

Comment:
It seems like the `@author` tag got lost here :smile:
Sorry if there was something confusing, please add it back, with lowercase `@author`